assets/src/blocks/Covers/CoversEditor.js

```diff
@@ -82,10 +82,11 @@ const renderEdit = (attributes, toAttribute, setAttributes) => {
           <div>
             <label>{__('Manual override', 'planet4-blocks-backend')}</label>
             <PostSelector
-              value={posts}
+              label={__('Select articles', 'planet4-blocks-backend')}
```

Comment:
Is `articles` the best word to use here? 🤔 Maybe better to use `pages` or `posts` (same on line 89)